src/components/Button/Button.jsx

```diff
@@ -39,96 +40,99 @@ const Button = forwardRef(({ children, icon, direction, caption, captionColor, d
   </Base>
 ))
 
-const colorVariants = variant({
-  default: 'filled',
-  key: 'button',
-  variants: {
-    filled: css`
-      background-color: ${({ color }) => th.color(color)};
-      border-color: ${({ color }) => th.color(color)};
-      color: white;
-      &:hover {
-        background-color: ${({ color }) => th.color(`${color}_hover`)};
-        border-color: ${({ color }) => th.color(`${color}_hover`)};
-      }
-      &:active {
-        background-color: ${({ color }) => th.color(`${color}_active`)};
-        border-color: ${({ color }) => th.color(`${color}_active`)};
-      }
-      &:disabled {
-        background-color: disabled;
-        border-color: disabled;
-        color: white;
-      }
-    `,
-    outlined: css`
-      ${StyledIcon} {
-        fill: ${({ color }) => th.color(color)};
-        &:hover {
-          fill: ${({ color }) => th.color(`${color}_hover`)};
+const colorVariants = ({ theme: { colors }, color }) => {
+  return variant({
+    prop: 'variant',
+    variants: {
+      filled: {
+        backgroundColor: `${colors[`${color}`]}`,
```

Comment:
Nesse caso nao precisa do template string "de fora", removendo ficaria assim:
```suggestion
        backgroundColor: ${colors[`${color}`]},
```

**obs**: repetir nos outros lugares que utilizem essa sintaxe

Reply:
nesse caso em especifico em que a estamos passando os estilos dentro de um objeto, nem precisamos da template string:
```backgroundColor: colors[color]```